lib/utils/memoize.js

```diff
@@ -3,13 +3,15 @@
 const calculateCacheKeyForTree = require('calculate-cache-key-for-tree');
 const CACHE = Object.create(null);
 
-module.exports = function memoize(func) {
-  return function _memoize() {
+function memoize(func) {
```

Comment:
Why switch this from a default export to a named export? Other util modules seem to use default exports.

Reply:
We need this export to mock this on tests with sinon. And we already have this export here - https://github.com/ember-engines/ember-engines/blob/master/lib/utils/append-compact-reexports-if-needed.js#L15

Reply:
@dgeb can we merge this?